ironfish/src/mining/stratum/messages.ts

```diff
@@ -44,6 +44,24 @@ export type MiningNotifyMessage = {
   header: string
 }
 
+export type MiningGetStatusMessage = {
+  publicAddress?: string
+}
```

Comment:
Idk if we want to make this similar to MiningDisconnectMessage when we have no required arguments, or just force all messages to have a payload. I'm fine either way.

```suggestion
export type MiningGetStatusMessage = {
  publicAddress?: string
} | undefined
```